spec/adapters/googlecharts_spec.rb

```diff
@@ -30,5 +30,9 @@
     it "Table class must be GoogleVisualr::DataTable " do
       expect(Daru::View::Table.new.table).to be_a GoogleVisualr::DataTable
     end
+    it "Table class must be GoogleVisualr::DataTable when data objects are" \
+       " of class Daru::Vector" do
+      expect(Daru::View::Table.new(@data_vec1, @options).table).to be_a GoogleVisualr::DataTable
+    end
```

Comment:
Please also verify the `data` and `options` attribute in `Daru::View::Table` class.